packages/apps/transfer/.eslintrc.js

```diff
@@ -2,7 +2,7 @@ require('@rushstack/eslint-config/patch/modern-module-resolution');
 
 module.exports = {
   plugins: ['jest'],
-  extends: ['next/core-web-vitals', '@kadena-dev/eslint-config/profile/react'],
+  extends: ['@kadena-dev/eslint-config/profile/next'],
   parserOptions: { tsconfigRootDir: __dirname },
-  ignorePatterns: ['**/*.spec.ts', '**/*.spec.tsx', '**/generated/**'],
+  ignorePatterns: ['**/generated/**'],
```

Comment:
Does removing this still work with VSCode ESLint Plugin? I had issues with the base config not being able to resolve the plugins from next

Reply:
yes this is totally fine since we added tests on purpose to eslint ignore just for transfer since we had some difficulties, but now they are resolved and there's no need to ignore tests anymore, just like other next setups in monorepo